CHANGELOG.md

```diff
@@ -2674,26 +2677,35 @@ Additionally, thanks to [Alexander Turek][@derrabus] for consulting on the repo
 - Fixed an issue where excluding a file using a @codingStandardsIgnoreFile comment would produce errors
     - For PHPCS, it would show empty files being processed
     - For PHPCBF, it would produce a PHP error
-- Fixed bug #1233 : Can't set config data inside ruleset.xml file
-- Fixed bug #1241 : CodeSniffer.conf not working with 3.x PHAR file
+- Fixed bug [#1233] : Can't set config data inside ruleset.xml file
+- Fixed bug [#1241] : CodeSniffer.conf not working with 3.x PHAR file
+
+[#1251]: https://github.com/squizlabs/PHP_CodeSniffer/issues/1251
+[#1270]: https://github.com/squizlabs/PHP_CodeSniffer/issues/1270
+[#1233]: https://github.com/squizlabs/PHP_CodeSniffer/issues/1233
+[#1241]: https://github.com/squizlabs/PHP_CodeSniffer/issues/1241
```

Comment:
I think it would be a good idea to keep these link lists in numeric order to prevent duplicate link definitions being added. Not much of an issue with short lists like the above, but it could be an issue for longer lists.
(here and elsewhere in this PR - I haven't noticed them not being in numeric order in previous PRs)

Reply:
Up to this point, I generated the list of issues in the order in which they appear in the changelog section for a particular version. I just pushed a new commit ordering the list of issues.

I will check the other lists added in previous PRs to see if any are out of numeric order, and if necessary, I will open a new PR to fix them.

Reply:
Thanks! I'll be adding a markdown linter in CI at some point, which should also help prevent duplicate links from being added.